src/file/format/mod.rs

```diff
@@ -93,18 +96,18 @@ lazy_static! {
 impl FileFormat {
     // TODO: pub(crate)
     #[doc(hidden)]
-    pub fn extensions(self) -> &'static Vec<&'static str> {
+    pub fn extensions(&self) -> &'static Vec<&'static str> {
```

Comment:
There's a `TODO: pub(crate)` here, that could be resolved right away, right? :smile:

Reply:
Sure, I'll see what I can do

Reply:
Required some changes in the example, but it is all pub (crate) now.